### ISSUE TITLE
profile_page in schema may be unnecessary

### ISSUE THREAD

**Issue by dkundathagard** [avengers5531/db_powon#12]
Since a "page" as we define it in our schema is just a container for posts, we don't actually have to query the member page itself, just the posts that belong to it. The profile_page relation could be replaced with a page_id field and a page_access field in member.

**Comment by xmithd** [comment]
I agree with. Feel free to make the changes. You may have to synchronize with @srikanth121 for the extra fields in the Member entity.
